pages/api/sendConfirmationEmail.ts

```diff
@@ -0,0 +1,83 @@
+import nodemailer from 'nodemailer'; // Nodemailer
+
+/**
+ * Send email to user to confirm appliction was received
+ */
+const sendConfirmationEmail = (to: string): Promise<void> => {
+  return new Promise((resolve, reject) => {
+    nodemailer
+      .createTransport({
+        host: process.env.NA_EMAIL_HOST as string,
+        port: parseInt(process.env.NA_EMAIL_PORT as string, 10),
+        auth: {
+          user: process.env.NA_EMAIL_USER as string,
+          pass: process.env.NA_EMAIL_PASSWORD as string,
+        },
+      })
+      .sendMail(
+        {
+          to,
+          // TODO: Use different email
+          from: process.env.NA_EMAIL_FROM,
+          subject: 'RCD Application Received',
+          text: text,
+          html: html,
+        },
+        error => {
+          return error ? reject(error) : resolve();
+        }
+      );
+  });
+};
+
+// TODO: Update email content
```

Comment:
Is this the updated copy?

Reply:
Nope! I was thinking of doing that in a separate PR